docker-compose.yaml

```
@@ -0,0 +1,49 @@
+services:
+  node:
+    build: .
+    environment:
+      PORT: 4000
+      ROUTER_URL: http://router:4000/
+      UNLEASH_URL: http://unleash:4242/api/
+      UNLEASH_API_TOKEN: default:development.unleash-insecure-api-token
+    ports:
+      - 4000:4000
+    volumes:
+      - ./router:/dist
+      - ./src:/web/src
+    command: yarn nodemon -L src/index.js
+    depends_on:
+      - unleash
```

```suggestion
    depends_on:
      - unleash:
          condition: service_healthy
```
https://github.com/compose-spec/compose-spec/blob/master/spec.md#long-syntax-1

Might be possible to avoid the "unable to connect" by adding a condition on the service being healthy.

---

that seemed to do it!